### PR TITLE
Updated dependencies

### PR DESCRIPTION
**Pull request by lhstrh**

### CODE REVIEW

**Comment by lhstrh** [comment]
Looks like this requires upgrading to Node 18.

**Comment by petervdonovan** [comment]
I am not sure why this is failing in CI since it does build locally with node 18.12.0.

File: .github/workflows/deploy-website.yml [hunk 1]
```diff
@@ -17,7 +17,7 @@ jobs:
           submodules: recursive
       - uses: actions/setup-node@v2
         with:
-          node-version: "16.x"
+          node-version: "18.x"
```
Comment (lhstrh) [inline .github/workflows/deploy-website.yml:20]:
```suggestion
          node-version: "19.x"
```

**Comment by axmmisaka** [comment]
Closing this because we changed platform.

For your curiosity, in Nov 2022 the old VS Code website platform only support Node 14, so upgrading from node 18 to node 19 is not a wise move 😛

**Pull request closed without merging**